doc/DEMO.md

```diff
@@ -245,6 +252,47 @@ cargo build
 
 Now we need to setup `revaultd` three times, one for each entity.
 
+#### Generating the Miniscript descriptors
+All wallets are going to track the same coins, which are specified using the [output descriptors language](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md).
+
+Since we want all wallets to be using the same descriptors, we are going to generate them once and for all.
+For this purpose you can use the `mscompiler` tool:
+```
+cd contrib/tools/mscompiler
+cargo build
+```
+
+This small command line tool takes 6 parameters as input, allowing to tweak the configuration of the
+Revault setup at the chain level. These tweakable parameters are:
+- The xpubs of the stakeholders
+- The raw public keys of the cosigning servers (as many as stakeholders xpubs)
+- The xpubs of the managers, and the threshold of the multisig (eg you may have 3 xpubs and a
+  threshold of 2 in order to allow 2 managers to Spend while the third one is on vacation)
+- The xpubs of the managers' "CPFP wallet". This functionality isn't implemented yet so you can put
```

Comment:
"Put garbage here" is not very helpful for the user :joy: Let's make the user generate one CPFP key to be put here?